src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -107,6 +108,7 @@ class MediaBaseViewer extends BaseViewer {
             .getPromise()
             .then(() => {
                 this.mediaEl.src = this.mediaUrl;
+                this.checkAutoplay();
```

Comment:
Added this after the src is set because doing so can disrupt play. Noted here: https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/